src/barplot.ts

```diff
@@ -247,6 +247,15 @@ export function drawLabels(size:number[], data:INumericalVector, labels: d3.Sele
     'display': (rowHeight >= 15) ? 'inline' : 'none',
     'font-size': '14px'
   });
+  const categories = (<any>data.desc.value).categories;
+  const nameToLabel = new Map<string, string>();
+  if (categories) {
```

Comment:
if the `data.desc.type === 'categorical' ` then the `.categories` have to exist and you know that the description is a `ICategoricalValueTypeDesc` - no need for any cast